src/frameworks/web/components/atoms/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import Tag from 'frameworks/web/components/atoms/Tag/Tag';
+import theme from 'theme';
+
+export default {
+  title: 'Tag',
+  decorators: [withKnobs],
+};
+
+export const DefaultTag = () => {
+  const label = text('Text', '태그');
+
+  const colorList = {
+    'Azuru' : theme.color.primary.Azure,
+    'Scarlet' : theme.color.primary.Scarlet,
+    'Moon' : theme.color.secondary.Moon,
+    'Failure' : theme.color.alert.Failure,
+    'Success' : theme.color.alert.Success,
+  }
+  const color = select('Color', colorList, theme.color.secondary.Moon);
+  return <Tag color={color} label={label}></Tag>
+}
+
+export const AcceptTag = () => {
+  const label = text('Text', '승인완료');
```

Comment:
동어 반복의 느낌이 있어서, 단순 "승인"이 더 좋을 것 같습니다.

```suggestion
  const label = text('Text', '승인');
```

Reply:
제플린 보면서 그대로 작성하느라 생각하지 못했네요. 수정했습니다!